test/unit_tests/apps/owning-a-home/js/explore-rates/RateCheckerChart-spec.js

```diff
@@ -0,0 +1,98 @@
+const BASE_JS_PATH = '../../../../../../cfgov/unprocessed/apps/owning-a-home/';
+const RateCheckerChart = require( BASE_JS_PATH + 'js/explore-rates/RateCheckerChart' );
+
+const HTML_SNIPPET = `
+  <section id="chart-section" class="chart">
+
+    <figure class="data-enabled loading">
+        <div id="chart" class="chart-area"></div>
+        <figcaption class="chart-caption">
+            <div class="caption-title">
+                Interest rates for your situation
+            </div>
+            <div class="rc-data-link">
+                <a href="#about" class="u-link-underline">
+                  About our data source
```

Comment:
Minor, should be four space indentation to match the rest.

Reply:
Fixed